index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <title>Tribute Page</title>
+  <link rel="stylesheet" href="style.css"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+  <main id="main">
+    <h1 id="title">Diego Armando Maradona</h1>
```

Comment:
- [ ] OPTIONAL: You could wrap this h1 tag inside in a header tag to make your code clean! 🧹